server/zanata-frontend/src/frontend/app/components/RBButton/story.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { Button } from 'react-bootstrap'
+
+storiesOf('Button', module)
```

Comment:
@davidmason I added this file when I was trying to create the combined editor/frontend storybook. 

You can go ahead and remove this (/RBButton).